tests/test_regression.py

```diff
@@ -485,6 +485,17 @@ def test_30_remove(self):
 
         expected = ['5.150.145.0/24', '109.161.64.0/20']
         self.assertEqual(expected, [n.prefix for n in tree])
+		
+	def test_31_covering_with_delete_error(self):
+        tree = radix.Radix()
+        tree.add('91.187.124.0/24')
+        tree.add('91.187.125.0/24')
+        tree.add('91.187.124.0/23')
+        tree.delete('91.187.124.0/23')
+        tree.add('91.187.124.0/23')
```

Comment:
Ah, I see, it's the seconds `.add` that causes the problem. I fixed the `expected` value in the test in this commit: https://github.com/mjschultz/py-radix/commit/e91e96e7ed21e346c6b566851dc45672526d5ca2

And then pulled your changes in and made a few more (unrelated) tweaks to the tests file in this PR: https://github.com/mjschultz/py-radix/pull/22.

Once Travis runs the tests I'll merge that and cut a new release.